tests/api.c

```diff
@@ -7481,13 +7593,40 @@ static void test_wolfSSL_set_options(void)
     AssertTrue((SSL_set_options(ssl, SSL_OP_NO_COMPRESSION) &
                                SSL_OP_NO_COMPRESSION) == SSL_OP_NO_COMPRESSION);
 
+		AssertTrue(SSL_set_msg_callback(ssl, msg_cb) == SSL_SUCCESS);
+		SSL_set_msg_callback_arg(ssl, arg);
+
+		AssertTrue(SSL_CTX_set_alpn_protos(ctx, protos, len) == SSL_SUCCESS);
+
     SSL_free(ssl);
     SSL_CTX_free(ctx);
 
     printf(resultFmt, passed);
     #endif /* defined(OPENSSL_EXTRA) && !defined(NO_CERTS) && \
              !defined(NO_FILESYSTEM) && !defined(NO_RSA) */
 }
+	  #if defined(OPENSSL_EXTRA) && !defined(NO_CERTS)
+static int verify_cb(int ok, X509_STORE_CTX *ctx)
+{
+	  (void) ok;
+		(void) ctx;
+		printf("ENTER verify_cb\n");
+		return SSL_SUCCESS;
+}
+#endif
+
+static void test_wolfSSL_X509_STORE_CTX(void)
+{
+	  #if defined(OPENSSL_EXTRA) && !defined(NO_CERTS)
+	  X509_STORE_CTX *ctx   = NULL ;
+
+    printf(testingFmt, "test_wolfSSL_X509_STORE_CTX(()");
+		AssertNotNull(ctx = X509_STORE_CTX_new());
+		X509_STORE_CTX_set_verify_cb(ctx, (void *)verify_cb);
```

Comment:
Does "verify_cb" ever get called in this unit test?  If not, we should add a test that would force the verify_cb to be called.  This will let us check to make sure it is correctly being registered and called.

Reply:
Looks good, as discussed.